### DISCUSSION TITLE
Help with Bed leveling

### DISCUSSION THREAD

**Discussion by sandgink** [mriscoc/Ender3V2S1#1373] [General]
Greetings All,

I have been using the mriscoc firmware on my E3V2 for a while, and I had to replace the mother board because I thought the extruder output had failed,(turned out to be a bad cable) lesson learned. Anyway I replaced it with a BTT SKR mini E3 V3,  installed the proper mriscoc firmware and can't get the bed leveling to work, I can activate it from the menu but when I use the G29 the printer ignores it.

here is my start code this is a copy paste from the web so if it needs to be corrected let me know.

thanks in advance

G90 ; use absolute coordinates
M83 ; extruder relative mode
M140 S[bed_temperature_initial_layer] ; set final bed temp
M104 S[nozzle_temperature_initial_layer] ; set final nozzle temp
G4 S10 ; allow partial nozzle warmup
G29 ; abl probe
G28 ; auto home  
G1 Z50 F240
G1 X2 Y10 F3000
M190 S[bed_temperature_initial_layer] ; wait for bed temp to stabilize
M109 S[nozzle_temperature_initial_layer] ; wait for nozzle temp to stabilize
G1 Z0.28 F240
G92 E0
G1 Y140 E10 F1500 ; prime the nozzle
G1 X2.3 F5000
G92 E0
G1 Y10 E10 F1200 ; prime the nozzle
G92 E0

![screen](https://github.com/user-attachments/assets/bae162b5-b58c-4b0b-9c6f-10fd2d1eedb8)

**Comment by mriscoc** [comment]
UBL -> G29 P1, Check the Wiki for more info
Also the order of your G-code in your example is incorrect.

**Reply by sandgink** [reply]
You are amazing and so is your work.  thank you soooo much.
